index.html

```diff
@@ -85,14 +85,26 @@ <h4 class="variableTitle">Variables</h4>
             <div class="vizCol">
                 <h4 class="graphsTitle">Graphs</h4>
                 <div class="dataViz">
-                <canvas id="firstGraph"></canvas>
-                <canvas id="secondGraph"></canvas>
+                    <div class="canvas-outer-bar">
+                        <div class="canvas-inner">
+                            <canvas id="firstGraph"></canvas>
+                        </div>
+                    </div>
```

Comment:
Here, I wrapped the canvas element with 2 div elements. This is purely to give the necessary hooks to add CSS that forces the canvas to scroll at really small viewports.

I created two different versions: canvas-outer-bar and canvas-outer-radar. Bar is wider to accommodate its size.